pkg/logs/tekton_logging.go

```diff
@@ -72,7 +73,7 @@ func GetTektonPipelinesWithActivePipelineActivity(jxClient versioned.Interface,
 			prBuildNumber = findLegacyPipelineRunBuildNumber(&pr)
 		}
 		paName := createPipelineActivityName(pr.Labels, prBuildNumber)
```

Comment:
TBH, I don't get what's happening in this method, but that's apart from the current changes